docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -156,3 +157,7 @@ To destroy the component:
 ```bash
 lokoctl component render-manifest prometheus-operator | kubectl delete -f -
 ```
+
+## What's next
+
+- Learn [how to access monitoring stack](../../how-to-guides/access-monitoring.md) installed using above tutorial.
```

Comment:
Is access the right word here ? How about `enable` ?

Reply:
It is right. Because this document describes how to install it, which IMO is the same as `enable` it.

Reply:
The new doc is about **accessing** the stack, no?